vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java

```diff
@@ -1099,18 +1099,22 @@ private void createAnalysisRecords(
                                                   case URL -> "url";
                                                 })
                                         .collect(Collectors.toList())))))));
-    context
-        .select(DSL.jsonObject(fields))
-        .from(WORKFLOW_RUN)
-        .where(condition)
-        .forEach(
-            result -> {
-              try {
-                jsonGenerator.writeRawValue(result.value1().data());
-              } catch (IOException e) {
-                throw new RuntimeException(e);
-              }
-            });
+    try {
+      context
+          .select(DSL.jsonObject(fields))
+          .from(WORKFLOW_RUN)
+          .where(condition)
+          .forEach(
+              result -> {
+                try {
+                  jsonGenerator.writeRawValue(result.value1().data());
+                } catch (IOException e) {
+                  throw new RuntimeException(e);
+                }
+              });
+    } catch (Exception e) {
```

Comment:
createAnalysisRecords throws SQLException, why are you catching and throwing RuntimeException here?